src/colvarbias_abf.cpp

```diff
@@ -841,7 +841,7 @@ cvm::real colvarbias_abf::calc_energy()
   if (num_variables() != 1) return 0.0;
   
   // Get the home bin.
-  int home0 = gradients->current_bin_scalar(0);
+  unsigned int home0 = gradients->current_bin_scalar(0);
   if (home0 < 0) return 0.0;
```

Comment:
If `home0` is unsigned, won't this return always `0.0`?

Reply:
Yes, this should be an `int`. Making this change.